lib/puma.rb

```diff
@@ -19,10 +19,16 @@ def self.stats_object=(val)
     @get_stats = val
   end
 
-  def self.stats
-    @get_stats.stats
+  def self.stats_hash
```

Comment:
FWIW I don't like the rename. We should either add stats_hash or just change stats to a hash and bump major.

Reply:
I was guessing this would land on a 4.x release, but happy to change :)

Reply:
Yeah no way for you to know really but I'm targeting the next release to be a major bump 👍